microprofile/WFLY-14940_upgrade-microprofile-health-3.1.adoc

```diff
@@ -0,0 +1,69 @@
+= Upgrade MicroProfile Health to 3.1
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,health,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+MicroProfile 4.1 provides a minor upgrade for https://github.com/eclipse/microprofile-health/releases/tag/3.1[MicroProfile Health 3.1] with changes described in its https://download.eclipse.org/microprofile/microprofile-health-3.1/microprofile-health-spec-3.1.html#release_notes_3_1[Release Notes].
+
+This component upgrade also covers the upgrade of smallrye-health 3.1.1 that implements MicroProfile Health 3.1.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-14940[WFLY-14040]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1763[EAP7-1763]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+[X] Engineering
+[ ] QE
+
```

Comment:
If I look at the [proposal template](https://github.com/wildfly/wildfly-proposals/blob/main/design-doc-template.adoc) I can see that three items are missing here in the `Issue Metadata` section, namely:

* Affected Projects or Components
* Other Interested Projects
* Relevant Installation Types

I think that all of these deserve to appear, specifically it is good to know what projects will be affected, what can be interested (add a N/A if none, but at least we have it defined in here) and what installation types are relevant for it- now that we have traditional server, Bootable JAR etc.
Can you please add them?

Reply:
Can be resolved (I don't have the grants for doing so), thanks.